.github/workflows/validation.yml

```diff
@@ -161,10 +161,22 @@ jobs:
         run: MVN_ARGS=-DskipTests REVISION=${{ env.BRANCH_NAME }} make build-jar build-linux
       - name: copy to .vib/linux-install/scripts
         run: cp -v out/*deb .vib/linux-install/scripts
+      - name: zip .vib/linux-install/scripts
+        run: zip -R linux-deb-test.zip .vib/linux-install/scripts
```

Comment:
```
zip -R linux-deb-test.zip .vib/linux-install
```
Because you need to send goss files, I mean the resources are the entire goss project